app/javascript/packs/Components/Body.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import Filters from "./Filters";
+import RestaurantList from "./RestaurantList";
+
+import styles from './Body.module.scss'
+
+export default class Body extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            isLoaded: false,
+            error: false,
+            visibleItems: [],
+            items: [],
+        }
+    }
+
+    componentDidMount() {
+        fetch("http://localhost:3000/restaurants")
+            .then(res => res.json())
+            .then(
+                (result) => {
+                    const transformedResults =
+                        result.map((item) => {
+                            item.max_delivery_time = Number.parseInt(item.max_delivery_time);
```

Comment:
I think there's no need to add the `Number.` before `parseInt`, it's the same as the global function.
[MDN recommends](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Description) on always adding `10` as the second parameter to: `parseInt(item.max_delivery_time, 10)`.